webapp/src/blocks.tsx

```diff
@@ -482,7 +490,10 @@ export class Editor extends toolboxeditor.ToolboxEditor {
             pxtblockly.external.setPromptTranslateBlock(dialogs.promptTranslateBlock);
         }
 
-        pxtblockly.external.setCopyPaste(copy, cut, this.pasteCallback, this.copyPrecondition, this.pastePrecondition);
+        // Disable clipboard overwrite (initCopyPaste) when accessible blocks enabled.
+        if (!data.getData<boolean>(auth.ACCESSIBLE_BLOCKS)) {
```

Comment:
you should register a data subscriber to listen to updates to the `auth.ACCESSIBLE_BLOCKS` setting. check out the constructor of the `Editor` in `monaco.tsx` for an example

Reply:
At the moment, we reload MakeCode when the keyboard navigation plugin is toggled on/off so I'm not sure this is needed, unless there is a way this setting can change other than the user interacting with the settings menu?

Reply:
Sign in and sign out both involve a page reload, and we only fetch user preferences from cloud on page load, so I think it's ok.